packages/components/src/slot-fill/types.ts

```diff
@@ -36,15 +36,21 @@ export type SlotComponentProps =
 
 			/**
 			 * A function that returns nodes to be rendered.
-			 * Not supported when `bubblesVirtually` is true.
+			 * Not supported when `bubblesVirtually` is `true`.
 			 */
 			children?: never;
 
 			/**
 			 * className.
-			 * Not supported when `bubblesVirtually` is true.
+			 * Not supported when `bubblesVirtually` is `false`.
```

Comment:
Nitpicking, but I think it's more readable when we inverse it:

```suggestion
			 * Supported only when `bubblesVirtually` is `true`.
```

If you agree, feel free to update the rest of the similar comments in this file.